src/System.Private.CoreLib/src/System/Threading/ClrThreadPool.cs

```diff
@@ -10,20 +10,22 @@ namespace System.Threading
     /// <summary>
     /// A thread-pool run and managed on the CLR.
     /// </summary>
-    internal static partial class ClrThreadPool
+    internal partial class ClrThreadPool
     {
+        public static ClrThreadPool s_threadPoolInstance = new ClrThreadPool();
```

Comment:
Singletons are typically named Instance like a property even though it's a read-only field